RetailCoder.VBE/Inspections/QuickFixes/AssignedByValParameterQuickFixHelper.cs

```diff
@@ -0,0 +1,195 @@
+﻿using Antlr4.Runtime;
+using Antlr4.Runtime.Tree;
+using Rubberduck.Parsing.Grammar;
+using Rubberduck.Parsing.Symbols;
+using Rubberduck.VBEditor;
+using Rubberduck.VBEditor.SafeComWrappers.Abstract;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+
+namespace Rubberduck.Inspections.QuickFixes
+{
+    class AssignedByValParameterQuickFixHelper
+    {
+        private Declaration _target;
+        private QualifiedSelection _selection;
+        private HashSet<string> _namesInUse;
```

Comment:
I'm new to C#, but my initial checks online and simply typing "Set" into VS indicate that Set does not exist in C#. (?) HashSet was selected to create a list/set of unique values.  If I'm mistaken about "Set", I am happy to change this.